### ISSUE TITLE
[task] HLR | Gather form update requirements

### ISSUE THREAD

**Issue by eileen-coforma** [department-of-veterans-affairs/va.gov-team#82532]
## Value Statement

**_As a_** product team

**_I want to_** gather the requirements for the 0996 form updates, and understand which part of the form updates are relevant to the VA.gov online form

**_So that_** we can make specific updates to the VA.gov 0996 form

---

## Background Context

There's been an update to the paper 20-0996 Higher Level Review form. The new form contains changes to the instructions and content of the form fields. The new paper form is live. Before we start development, we need to identify what our business stakeholders need with these updates.

## Outcome, Success Measure, KPI(S), and Tracking Link

- Add here

## Acceptance Criteria

- [x] SMEs have been Identified for review of this work
  - Shireen Lackey
  - Monica Willis
  - Alejandro Mendiolaflores
- [x] Changes have been identified for the online form
- [x] Stakeholder requirements have been captured
  - [x]  The impact that the business logic has on claim development is captured
  - [x]  The meaning of content is clear
- [x] Align with OCTO stakeholders on the work

## Designs and Build Notes

- [MURAL](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1714761005965/cf4071f13c521f01c515757de3ab6e3e87ba22ba?sender=ud193ce75f65a17a316ea6194) - Current form screens + updated PDF changes

### Enablement team (if needed)
@jstrothman 

## Out of scope

- Add here

### Open questions about form changes
- Do we still include the _area of disagreement_?
  - Do we need to remove the examples of area of disagreement?
- Do we need to adjust the AMA opt-in?
- Do we need to include a way to add additional explanation (in lieu of informal conference)?
  - How does including an additional explanation affect the way the form gets processed?
- Does the Veteran need to submit a VA Form 21-22 and VA Form 21-22a before they submit an HLR if they want to include their representative?
  - Can the forms be submitted concurrent to the 0996? 
- Is the additional information about why issues aren't pre-loading still relevant and accurate? 

## Tasks

- [x] Set meeting with SMEs to go through form changes
- [x] Capture update requirements
- [x] Align on scope of work

## Definition of Ready
- [x] Final approved form is available
- [x] List of collaborating SMEs are identified
- [x] Mural board for facilitation is ready
- [x] Session scheduled with for SMEs and enablement team

## Definition of Done
- [x] Requirements captured
- [ ] Scope defined
- [x] SMEs identified

**Comment by eileen-coforma** [comment]
@saderagsdale 

**Questions about scope**
- Are we including the unauthenticated pages within the scope of this? I assume so, but given the dependence on the content team (and possible inability to embed cuz of overload), do we want to take suggesting changes on as part of the updates?
- Do we need an initiative brief for this work, and should it be included within this AC?

- Is this [Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1714761005965/cf4071f13c521f01c515757de3ab6e3e87ba22ba?sender=ud193ce75f65a17a316ea6194) enough to get the conversation started?

If so, I'm ready to discuss, but may need more time if I'm also breaking down the unauthenticated page.

**Comment by eileen-coforma** [comment]
Session scheduled for this afternoon.

**Comment by skylerschain** [comment]
### Answers to some of the above questions from 5/30 meeting

[Mural notes](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1714761005965/cf4071f13c521f01c515757de3ab6e3e87ba22ba?sender=u80fb5a3b8f7951f35d701197)

Do we still include the area of disagreement?
> We should keep it

Do we need to remove the examples of area of disagreement?
> No

Do we need to include a way to add additional explanation (in lieu of informal conference)?
> (If this means "Written statement") then no.

How does including an additional explanation affect the way the form gets processed?
> It sounds like limiting the exposition Veterans add is helpful for keeping the claim from getting bogged down or triggering the need for a supplemental claim due to additional evidence being added.

Does the Veteran need to submit a VA Form 21-22 and VA Form 21-22a before they submit an HLR if they want to include their representative?
> No, it can be submitted at the same time. But in order for VA to contact rep, there will need to be a 21-22/22a submitted.

Can the forms be submitted concurrent to the 0996?
> 21-22/22a can yes

### Questions we haven't yet answered

Is the additional information about why issues aren't pre-loading still relevant and accurate?

Are the examples of disagreement in Section 2 still valid?

Do we need to adjust the AMA opt-in?

**Comment by skylerschain** [comment]
Sent email to Shireen and Alejandro detailing the remaining open questions. Hope to hear back from Shireen when she is back from PTO next week (June 10).
